### ISSUE TITLE
The International Date Line issue

### ISSUE THREAD

**Issue by vyskocil** [Framstag/libosmscout#901]
If I remember well some issues were already fixed around the 180° longitude line but there remain some.
With libosmscout the world background map is drawn like this : 

![IMG_0382](https://user-images.githubusercontent.com/452704/89308273-59d57a80-d672-11ea-910f-89b5c59c619f.jpg)
![IMG_0382](https://user-images.githubusercontent.com/452704/89308283-5d690180-d672-11ea-8a2f-90436ab622dc.jpg)

There are missing land parts. 
On OSM the world map is drawn tiled like this : 

![Screenshot 2020-08-04 at 16 42 23](https://user-images.githubusercontent.com/452704/89308487-9d2fe900-d672-11ea-9993-9fb16d063a53.png)
![Screenshot 2020-08-04 at 16 51 08](https://user-images.githubusercontent.com/452704/89308593-bb95e480-d672-11ea-9beb-6aaf1b0547b2.png)

Maybe we may also do it like OSM ?

**Comment by Karry** [comment]
> There are missing land parts.

I don't think so. There is just one extra "sea tile" column on the east (`<0, n>`). It should be `<0, n-1>` :-)

**Comment by Framstag** [comment]
Should this ticket still be open or is it OK to close it? If we should keep it open, what is missing?

**Comment by Karry** [comment]
you can close it IMO, but @vyskocil created it...

**Comment by Framstag** [comment]
@vyskocil Is this fixed for you? Can we close it?

**Comment by vyskocil** [comment]
It's almost right now but in the very south : 

![IMG_9218](https://user-images.githubusercontent.com/452704/95119996-2768fc00-074d-11eb-8210-3faf00299261.jpg)
![IMG_9217](https://user-images.githubusercontent.com/452704/95120011-2cc64680-074d-11eb-9414-b38cb2fe8ec1.jpg)

**Comment by Framstag** [comment]
@Karry, @vyskocil is this still an issue?

**Comment by vyskocil** [comment]
It's almost perfect but some artefacts at the junction : 

<img width="330" height="717" alt="Image" src="https://github.com/user-attachments/assets/b150bb28-43d9-4e39-ac10-d7ca1e442e88" />

**Comment by Framstag** [comment]
Do you have a reproducer (import plus some DrawMapXXX call)? Then I can let the AI find it out. I currently do not have time for manual analysis but have a rather cheap working AI setup.
